### PR TITLE
Update for Tomb of Sargeras

### PR DESCRIPTION
**Pull request by AlexAtTheBux**
* Keystones
  * Option to display verbose names ("Lower Karazhan (7)", "Darkheart Thicket (13)") or abbreviated names ("Kara L (7)", "DHT (13)")
  * Added spaces in "Mythic Keystone" and between the dungeon name and level

* Emissaries
  * Display a check mark if the character has already completed that quest or a question mark if the quest is ready to turn in
  * Added spaces between the faction name and the number of days remaining, and between the number and "days"
  * Default to "Unknown Emissary" if daily reset occurs and the available emissary quests aren't visible to the active character
    * This can be due to the character not having WQs unlocked or the character having already completed the emissaries prior to loading the addon.

* Raids
  * Enable display of Tomb of Sargeras lockouts
  * Temporary update for ToS LFR lockouts.
    * This will need to be adjusted over the course of the next 6-8 weeks as the LFR wings unlock and actual data becomes available
  * Add "altid"s for the three WoD raids. These should be the values for the LFR versions you enter from your Garrison.

### CODE REVIEW

**Comment by Stanzilla** [comment]
Nice! Can you drop the changes to `locale.lua` though, if we ever want to get this back on Curse, those packager replacements need to stay in tact.

Thanks!

File: SavedInstances/SavedInstances.lua [hunk 1]
```diff
@@ -2893,28 +2930,54 @@ function core:RefreshDailyWorldQuestInfo()
     local title = GetQuestLogTitle(GetQuestLogIndexByID(BountyInfo.questID))
     local timeleft = C_TaskQuest.GetQuestTimeLeftMinutes(BountyInfo.questID)
     local _, _, isFinish, questDone, questNeed = GetQuestObjectiveInfo(BountyInfo.questID, 1, false)
-    if not isFinish and timeleft then
+    if timeleft then
       if timeleft > 2880 then
-        t.DailyWorldQuest.days2 = {}
+        if t.DailyWorldQuest.days2 then else t.DailyWorldQuest.days2 = {} end
```
Comment (Stanzilla) [inline SavedInstances/SavedInstances.lua]: Not sure why you did the loop here, what was the issue you tried to fix? Did you mean to do `if not t.DailyWorldQuest.days2 then t.DailyWorldQuest.days2 = {} end`?

File: SavedInstances/SavedInstances.lua [hunk 2]
```diff
@@ -48,6 +49,36 @@ vars.Indicators = {
   BLANK = "None",
 }
 
+local KeystoneAbbrev = {
```
Comment (Stanzilla) [inline SavedInstances/SavedInstances.lua]: ideally we would like these to be translatable so: L["EoA"] etc

File: SavedInstances/SavedInstances.lua (hunk 1 above)
Comment (Stanzilla) [inline SavedInstances/SavedInstances.lua]: Eh, I just saw oscarub did this all over the place and you just matched his style, nvm then.

Comment (AlexAtTheBux) [inline SavedInstances/SavedInstances.lua]: ```if x then else y``` is the same as ```if not x then y```...I just don't like using ```not``` (only explanation I can come up with for that).  When I was writing that part of the code I was getting errors without that loop, though I don't recall what the final reason was.

**Comment by AlexAtTheBux** [comment]
I don't understand what you mean regarding the locale.lua file.  I literally just modified the one I already had from Curse, which had all of the strings in it.

File: SavedInstances/SavedInstances.lua (hunk 2 above)
Comment (AlexAtTheBux) [inline SavedInstances/SavedInstances.lua]: My mind thinks in lua and English, so I didn't think about localizing the abbreviations. Definitely agree though.

**Comment by stevietv** [comment]
latest import for me works great. Good job, this should be merged!

**Comment by AlexAtTheBux** [comment]
Awesome, glad I could help!  How do we go about getting it actually merged?  And how do I go about becoming a contributor?

**Comment by Stanzilla** [comment]
@AlexAtTheBux The curseforge packager needs the lines you removed to be able to pull in the latest translations from the translation tool it uses. As far as getting this version Curse and how to become a contributor you'd have to ask @eTzmNcbkrng what his plans are as this is his fork.

I will merge when the locale.lua changes get reverted, everything else seems fine.

**Comment by ceylina** [comment]
Actually the TOS lfr is wrong. There are 2 bosses in wing 1, 3 bosses in wing 2, 2 bosses in wing 3, and one boss in wing 4. There are not 10 bosses in TOS and it has already been confirmed to have the wings split this way

So it need to be 

[1494] = { total=3, base=1, parent=1527,altid=nil }, -- TOS1: The Gates of Hell
[1495] = { total=3, base=4, parent=1527,altid=nil, remap={ 1, 2, 3 } }, -- TOS2: Wailing Halls
[1496] = { total=2, base=7, parent=1527,altid=nil, remap={ 1, 2} }, -- TOS3: Chamber of the Avatar
[1497] = { total=1, base=9, parent=1527,altid=nil, remap={ 1 } }, -- TOS4: Deceiver's Fall


The remap might change as wings go live but all else is correct for how LFR will be laid out.

**Comment by Stanzilla** [comment]
Awesome, thank you!

**Pull request merged**